### PR TITLE
Fix/global projects register

### PR DESCRIPTION
**Pull request by otakumike**

### CODE REVIEW

File: initialize/main.go [hunk 1]
```diff
@@ -26,61 +26,38 @@ func TokConfig(command string) {
...
-
-	viper.SetDefault("Global.Syncservice", "docker")
-
-	gc := filepath.Join(h, ".tok/global.yml")
-
-	// Check if the global config file exist, and read it in if it does
-	utils.DebugString("checking for global config at $HOME/.tok/global.yml")
-	_, err = os.Stat(gc)
+	oc := filepath.Join(h, ".tok/config.yml")
+	_, err = os.Stat(oc)
```
Comment (yjimk) [inline initialize/main.go]: Better to use the `fs.Exists` method for this?

File: initialize/main.go [hunk 2]
```diff
@@ -26,61 +26,38 @@ func TokConfig(command string) {
...
+		fs.Remove(oc)
+		fmt.Println(Magenta("Tokaido has removed your legacy global config file in $HOME/.tok/config.yml. You don't need it anymore"))
 	}
 }
 
 func readProjectConfig(command string) {
 	utils.DebugString("reading project config")
 	pr := fs.ProjectRoot()
 
+	viper.SetDefault("Global.Syncservice", "docker")
```
Comment (yjimk) [inline initialize/main.go]: woo

File: services/docker/generate.go [hunk 3]
```diff
@@ -105,12 +105,15 @@ func UnmarshalledDefaults() conf.ComposeDotTok {
...
-		err = yaml.Unmarshal(dockertmpl.TokaidoDockerSiteVolumeAttach(conf.GetConfig().Tokaido.Project.Path), &tokStruct)
+		err = yaml.Unmarshal(dockertmpl.TokaidoDockerSiteVolumeAttach(conf.GetProjectPath()), &tokStruct)
 		if err != nil {
 			log.Fatalf("Error attaching persistent /tokaido/site volume: %v", err)
 		}
 
 	} else {
+		fmt.Println("CONF AT THIS STAGE::::\n")
+		fmt.Println(conf.GetConfig())
+		fmt.Println("\n\n\n\n")
```
Comment (yjimk) [inline services/docker/generate.go]: Looks like debug messages? Did you want to leave this in?

Comment (otakumike) [reply services/docker/generate.go]: ooops!

File: initialize/main.go (hunk 1 above)
Comment (otakumike) [reply initialize/main.go]: yup good point - have fixed

**Comment by otakumike** [comment]
> Touched a lot of files on this one, can we get Roland to test before merging in?

Easier to get it all tested in develop with everything else. I don't want to ask people to test too many things and have them lose patience.

**Pull request merged**